### PR TITLE
Add cleanup rules for french

### LINKED ISSUE #268: Add French Cleanup file
According to https://github.com/mozilla/voice-web/pull/2262#issuecomment-553848602 the EN cleanup does not work for FR. This should be changed. Here are the rules that are valid for French:

```
      .replace('  ', ' ')
      .replace(/^,+\s/, '') // we do not want commas at the beginning of the sentence
      .replace(/^,+/, '') // we do not want commas at the beginning of the sentence
      .replace(/,+$/, '') // we do not want commas at the end of the sentence
```

### PR DESCRIPTION
**Pull request by lissyx**
Fixes #268

### CODE REVIEW

**Comment by lissyx** [comment]
Green tests: https://travis-ci.com/Common-Voice/sentence-collector/builds/140473733

**Comment by lissyx** [comment]
@MichaelKohler Ready for review

File: test/shared/cleanup/languages/fr.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    'this is ! wrong',
+  ];
+
+  const computed = fr.clean(orig);
+
+  t.deepEqual(computed, expected);
+});
+
+/*
+      .replace(/([^ ]|^)\?/g, '$1 ?')
```
Comment (MichaelKohler) [inline test/shared/cleanup/languages/fr.js]: Is this TODO or just a leftover that can be deleted?

Comment (lissyx) [reply test/shared/cleanup/languages/fr.js]: That's what you get for doing 5 things at the same time. Leftover, I've removed :)

File: test/shared/cleanup/languages/fr.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    'this is : wrong',
+    'this is ; wrong',
+    'this is ; wrong',
+    'this is ; wrong',
+    'this is wrong ? ',
+    'this is ? wrong',
+    'this is ? wrong',
+    'this is ? wrong',
+    'this is ? wrong',
+    'this is wrong ! ',
```
Comment (MichaelKohler) [inline test/shared/cleanup/languages/fr.js]: So the sentence would end with a space?

Comment (lissyx) [reply test/shared/cleanup/languages/fr.js]: Good catch, thanks :)

**Pull request merged**